resources/Exercise2/bethany1.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Indent the code properly follow below video
https://www.youtube.com/watch?v=bB7MLP37zqk

Reply:
Hi Srinivas,
Did few changes . unable to add logo in exact position. I am working on it and will go through the topics which u sent yesterday.

![image](https://github.com/rmh444/Demoproject/assets/141698090/775e68e4-0f8d-4d46-aa87-3cae1de78231)